src/Styles/custom.scss

```diff
@@ -0,0 +1,116 @@
+@import "./variables";
+@import "./colors";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  //   outline: 1px solid red;
+}
+
+@mixin flex_center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.sidebar {
+  width: 100%;
+  max-width: 15rem;
+  min-width: 15rem;
+  height: 100%;
+  background: $sidebar;
+  position: absolute;
+  left: -50rem;
+  transition: left 0.5s ease-in-out;
+  @media screen and (min-width: 768px) {
+    position: relative;
+    left: 0;
+  }
+  &.active {
+    left: 0;
+  }
+  .close_icon {
+    position: absolute;
+    top: 0.5rem;
+    color: white;
+    right: 1rem;
+    border: 1px solid red;
+    padding: 0.5rem;
+    font-size: 1.2rem;
+    border-radius: 0.5rem;
+    display: block;
+    cursor: pointer;
+    @media screen and (min-width: 768px) {
+      display: none;
+    }
+  }
+
+  &_inner {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .logo {
+      width: 100%;
+      height: 6rem;
+      background: $primary;
+      @include flex_center();
+      gap: 1rem;
+      img {
+        width: 100%;
+        max-width: 3rem;
+        height: 80%;
+        object-fit: contain;
+      }
+      span {
+        font-size: 1.5rem;
+        font-weight: 600;
+      }
+    }
+  }
+
+  &_items {
+    margin-top: 1rem;
+    // flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    // background: #000;
+
+    &_inner {
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      width: 100%;
+
+      li {
+        font-size: 1.1rem;
+        font-weight: 500;
+        width: 100%;
+        height: 2.5rem;
+        @include flex_center();
+        justify-content: flex-start;
+        padding-inline: 1rem;
+        &:hover {
+          background-color: $primary;
+          color: $white;
+          cursor: pointer;
+          border-radius: 1rem;
+        }
+        .menu_icon {
+          margin-right: 0.5rem;
+        }
+      }
+    }
+  }
+}
+.menu_bar {
+  @include flex_center();
+  @media screen and (min-width: 768px) {
+    display: none;
+  }
+}
```

Comment:
why are you using external css is it because of ant design

Reply:
yes